frontends/common/options.cpp

```diff
@@ -64,13 +64,13 @@ CompilerOptions::CompilerOptions() : Util::Options(defaultMessage) {
                    [this](const char*) {
                        langVersion = CompilerOptions::FrontendVersion::P4_14;
                        return true; },
-                    "[Deprecated] Specify language version to compile",
+                    "[Deprecated] Specify language version to compile.",
```

Comment:
Should these be removed all together? How long have they been "deprecated"?

Reply:
the driver is using --std, can we use the same flag for all backends, instead of --p4v?